controllers/addresspool_controller.go

```diff
@@ -43,9 +41,12 @@ type AddressPoolReconciler struct {
 	Namespace string
 }
 
-const RetryPeriod = 5 * time.Minute
+const (
+	RetryPeriod                       = 5 * time.Minute
```

Comment:
Move this down to a minute?
It's long enough to not stress the api server too much, but short enough in case there's some transient error.
WDYT?

Reply:
so far we didn't really stress this to be sure if 1 or 5 will be better settings, I will drop it to 1min for now and tune further if needed